### PR TITLE
[rwc2023] serve_breakfast

### PR DESCRIPTION
**Pull request by shanki98**

### CODE REVIEW

File: challenge_serve_breakfast/src/challenge_serve_breakfast/navigate_to_and_pick_item.py [hunk 1]
```diff
@@ -112,4 +112,4 @@ def __init__(self, robot, pick_spot_id, pick_spot_navigation_area):
     rospy.init_node(os.path.splitext("test_" + os.path.basename(__file__))[0])
     robot_instance = get_robot("hero")
     robot_instance.reset()
-    NavigateToAndPickItem(robot_instance, "dinner_table", "in_front_of").execute()
+    NavigateToAndPickItem(robot_instance, "kitchen_table", "in_front_of").execute()
```
Comment (PetervDooren) [inline challenge_serve_breakfast/src/challenge_serve_breakfast/navigate_to_and_pick_item.py:115]: This fix was due to the entity id in bordeaux. We should keep this dinner table for in the lab

File: challenge_stickler_for_rules/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.0.2)
```
Comment (PetervDooren) [inline challenge_stickler_for_rules/CMakeLists.txt:1]: Filter per challenge
Dont include others

**Review by PetervDooren — CHANGES_REQUESTED** [review]

**Comment by MatthijsBurgh** [comment]
Please fix the conflicts...

File: challenge_stickler_for_rules/CMakeLists.txt (hunk 2 above)
Comment (MatthijsBurgh) [reply challenge_stickler_for_rules/CMakeLists.txt]: Fixing it is a pain in the *ss. I don't know who fucked up the git history

**Pull request merged**